lib/src/ui/movie_detail.dart

```diff
@@ -7,9 +7,9 @@ import '../models/result_movie.dart';
 class MovieDetail extends StatelessWidget {
   final ResultMovie result;
 
-  const MovieDetail({
+  const MovieDetail(
+    this.result, {
     Key? key,
-    required this.result,
```

Comment:
Why did you made this change?

Reply:
At first I had an error so I had solved it that way, but now, checking, I think you're right and it looks better as it was before, so I went back this change.